aws-lc-rs/scripts/run-rustls-integration.sh

```diff
@@ -2,17 +2,57 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0 OR ISC
 
-ROOT=$PWD
-RUSTLS_RING_STRING="^ring = \".*\""
-RUSTLS_OUR_STRING="ring = { path = \"${PWD}\", package = \"aws-lc-rs\"}"
-
-git clone https://github.com/rustls/rustls.git
-cd rustls
-
-# replace instances of ring with our crate
-if [[ "$(uname)" == "Darwin" ]]; then
-	find ./ -type f  -name "Cargo.toml" | xargs sed -i '' "s|${RUSTLS_RING_STRING}|${RUSTLS_OUR_STRING}|g"
-else
-	find ./ -type f  -name "Cargo.toml" | xargs sed -i "s|${RUSTLS_RING_STRING}|${RUSTLS_OUR_STRING}|g"
-fi
+ROOT="${GITHUB_WORKSPACE:-$(git rev-parse --show-toplevel)}"
+
+CLEANUP_ON_EXIT=()
+
+function cleanup() {
+    for x in ${CLEANUP_ON_EXIT[@]}; do
+        rm -rf "${x}"
+    done
+}
+
+trap cleanup EXIT
+
+# Get aws-lc-rs version
+pushd "${ROOT}/aws-lc-rs"
+AWS_LC_RS_VERSION=$(cargo read-manifest | jq .version -r)
+popd &>/dev/null # "${ROOT}/aws-lc-rs"
+
+RUSTLS_WEBPKI_DIR="$(mktemp -d)"
+CLEANUP_ON_EXIT+=("${RUSTLS_WEBPKI_DIR}")
+
+RUSTLS_DIR="$(mktemp -d)"
+CLEANUP_ON_EXIT+=("${RUSTLS_DIR}")
+
+git clone https://github.com/rustls/webpki.git "${RUSTLS_WEBPKI_DIR}"
+git clone https://github.com/rustls/rustls.git "${RUSTLS_DIR}"
```

Comment:
Instead of pulling from the rustls' repo, should be test against what they've published to crates.io?